modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TxAbstractTest.java

```diff
@@ -383,7 +402,7 @@ public void testRepeatedCommitRollbackAfterUpdateWithException() throws Exceptio
     }
 
     @Test
-    public void testRepeatedCommitRollbackAfterRollbackWithException() throws Exception {
+    public void testRepeatedCommitRollbackAfterRollbackWithException() {
```

Comment:
Let's also cover testDeleteUpsertAllRollback. It's the test with highest failure rate.